vzmem

```diff
@@ -12,8 +12,8 @@ my $TOP = `top -b -n1 -d0`;
 my $PAGE_SIZE_KB = 4;
 my $MAX_WORD = (`uname -m` =~ /64/? "9223372036854775807" : "2147483647");
 my ($DIR) = grep { -d $_ } ("/etc/sysconfig/vz-scripts", "/etc/vz/conf");
-my ($MEM_KB) = $TOP =~ /^Mem:\s*(\d+)k/mi or die "Cannot parse 'top' command!\n";
-my ($SWP_KB) = $TOP =~ /^Swap:\s*(\d+)k/mi or die "Cannot parse 'top' command!\n";
+my ($MEM_KB) = $TOP =~ /^KiB Mem:\s*(\d+)/mi or die "Cannot parse 'top' command!\n"; 
```

Comment:
***top***:
```
[root@centos6 ~]# top -b -n1 -d0
...
Mem:    262144k total,    46156k used,   215988k free,        0k buffers
Swap:   524288k total,        0k used,   524288k free,    29472k cached
```

```
[root@debian7 ~]# top -b -n1 -d0
...
KiB Mem:    440104 total,   176584 used,   263520 free,        0 buffers
KiB Swap:        0 total,        0 used,        0 free,    23040 cached
```

***free***:
```
[root@centos6 ~]# top -b -n1 -d0
             total       used       free     shared    buffers     cached
Mem:        262144      45904     216240       3076          0      29480
-/+ buffers/cache:      16424     245720
Swap:       524288          0     524288
```

```
[root@debian7 ~]# top -b -n1 -d0
             total       used       free     shared    buffers     cached
Mem:        440104     176256     263848          0          0      23044
-/+ buffers/cache:     153212     286892
Swap:            0          0          0
```

As you can see, the outputs of ***top*** commands is different for centos6 and debian7, but they are same for ***free*** command.